appinventor/blocklyeditor/src/blocks/math.js

```diff
@@ -1056,6 +1066,7 @@ Blockly.Blocks['math_convert_number'] = {
   helpUrl: function() {
       var mode = this.getFieldValue('OP');
       return Blockly.Blocks.math_convert_number.HELPURLS[mode];
+//      return Blockly.Blocks.math_convert_number.HELPURLS()[mode];
```

Comment:
If this line isn't being used, we should just take it out.

Reply:
OK, this is done.